From 7220de95783ce262abc805779c76f727f86ae0e0 Mon Sep 17 00:00:00 2001
From: Martin Mares <mj@ucw.cz>
Date: Thu, 22 Jan 2015 22:44:49 +0100
Subject: [PATCH] Fixed memory allocation bug in name cache path code

---
 lib/names-cache.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/names-cache.c b/lib/names-cache.c
index c97ea30..ba5de6d 100644
--- a/lib/names-cache.c
+++ b/lib/names-cache.c
@@ -38,7 +38,7 @@ static char *get_cache_name(struct pci_access *a)
 
   buf = pci_malloc(a, strlen(pw->pw_dir) + strlen(name+1) + 1);
   sprintf(buf, "%s%s", pw->pw_dir, name+1);
-  pci_set_param_internal(a, "net.cache_name", buf, 0);
+  pci_set_param_internal(a, "net.cache_name", buf, 1);
   pci_mfree(buf);
   return pci_get_param(a, "net.cache_name");
 }
-- 
1.9.0

