diff -Naur pciutils-3.3.0/lib/Makefile pciutils-3.3.0.tpg/lib/Makefile
--- pciutils-3.3.0/lib/Makefile	2014-11-01 17:04:20.000000000 +0000
+++ pciutils-3.3.0.tpg/lib/Makefile	2015-01-24 14:52:26.905745240 +0000
@@ -57,9 +57,9 @@
 CFLAGS += -fPIC -fvisibility=hidden
 $(PCILIB): $(addsuffix .o,$(OBJS))
  ifdef PCI_HAVE_PM_DARWIN_DEVICE
-	$(CC) -shared $(LDFLAGS) $(SONAME) -Wl,-install_name,$(LIBDIR)/$(PCILIB) -o $@ $^ $(LIB_LDLIBS)
+	$(CC) $(LDFLAGS) -shared $(SONAME) -Wl,-install_name,$(LIBDIR)/$(PCILIB) -o $@ $^ $(LIB_LDLIBS)
  else
-	$(CC) -shared $(LDFLAGS) $(SONAME) -Wl,--version-script=libpci.ver -o $@ $^ $(LIB_LDLIBS)
+	$(CC) $(LDFLAGS) -shared $(SONAME) -Wl,--version-script=libpci.ver -o $@ $^ $(LIB_LDLIBS)
  endif
 endif
 
