diff --git a/lib/configure b/lib/configure
index bce4759..2bf0574 100755
--- a/lib/configure
+++ b/lib/configure
@@ -82,6 +82,8 @@ cat >$c <<EOF
 #define PCI_ARCH_SH
 #elif defined(__arm__)
 #define PCI_ARCH_ARM
+#elif defined(__aarch64__)
+#define PCI_ARCH_ARM
 #else
 #error Unknown Arch
 #endif
@@ -277,4 +279,4 @@ echo >>$m 'PCILIBPC=$(LIBNAME).pc'
 
 echo >>$c "#define PCILIB_VERSION \"$VERSION\""
 echo >>$cm "#define PCILIB_VERSION \"$VERSION\""
-sed '/"/{s/^#define \([^ ]*\) "\(.*\)"$/\1=\2/;p;d;};s/^#define \(.*\)/\1=1/' <$cm >>$m
\ No newline at end of file
+sed '/"/{s/^#define \([^ ]*\) "\(.*\)"$/\1=\2/;p;d;};s/^#define \(.*\)/\1=1/' <$cm >>$m
