diff -up pciutils-3.9.0/Makefile.4~ pciutils-3.9.0/Makefile
--- pciutils-3.9.0/Makefile.4~	2022-11-21 01:45:29.925501173 +0100
+++ pciutils-3.9.0/Makefile	2022-11-21 01:46:06.002721246 +0100
@@ -140,7 +140,7 @@ TAGS:
 	find . -name '*.[hc]' -exec etags --append {} +
 
 clean:
-	rm -f `find . -name "*~" -o -name "*.[oa]" -o -name "\#*\#" -o -name TAGS -o -name core -o -name "*.orig"`
+	rm -f `find . -name "*~" -o -name "*.o" -o -name "\#*\#" -o -name TAGS -o -name core -o -name "*.orig"`
 	rm -f update-pciids lspci$(EXEEXT) setpci$(EXEEXT) example$(EXEEXT) lib/config.* *.[578] pci.ids.gz lib/*.pc lib/*.so lib/*.so.* lib/*.dll lib/*.def lib/dllrsrc.rc *-rsrc.rc tags
 	rm -rf maint/dist
 
