diff -Naur pciutils-3.6.1/Makefile pciutils-3.6.1.tpg/Makefile
--- pciutils-3.6.1/Makefile	2018-07-27 17:36:23.728672000 +0000
+++ pciutils-3.6.1.tpg/Makefile	2018-07-27 17:37:34.150674915 +0000
@@ -111,7 +111,7 @@
 	find . -name '*.[hc]' -exec etags --append {} +
 
 clean:
-	rm -f `find . -name "*~" -o -name "*.[oa]" -o -name "\#*\#" -o -name TAGS -o -name core -o -name "*.orig"`
+	rm -f `find . -name "*~" -o -name "*.[o]" -o -name "\#*\#" -o -name TAGS -o -name core -o -name "*.orig"`
 	rm -f update-pciids lspci setpci example lib/config.* *.[78] pci.ids.* lib/*.pc lib/*.so lib/*.so.* tags
 	rm -rf maint/dist
 
